src/ts/store.tsx

```diff
@@ -4,6 +4,8 @@ export type ComponentType<P> =
   | React.ComponentClass<P>
   | React.StatelessComponent<P>;
 
+
+//  @internal
```

Comment:
Must be a doc comment.

```suggestion
/**
 * @internal 
 */
```